lib/src/Base/Func/EnumerateFunctionImplementation.cxx

```diff
@@ -66,7 +66,12 @@ Indices EnumerateFunctionImplementation::operator() (const UnsignedInteger index
 /* The inverse of the association */
 UnsignedInteger EnumerateFunctionImplementation::inverse(const Indices & indices) const
 {
-  throw NotYetImplementedException( HERE ) << "In EnumerateFunctionImplementation::inverse(const Indices & indices) const";
+  UnsignedInteger index = 0;
```

Comment:
Why not, but can't be the default implementation for NormInfEnumerateFunction as it is terribly inefficient in the general case (and inverse() is almost trivial in the case of NormInfEnumerateFunction)

Reply:
I did not found a direct way to compute the inverse, any clues here ?